assignment1/q2.cpp

```diff
@@ -0,0 +1,33 @@
+struct node{
+    int val;
+    node* next;
+};
+
+node *kth_element(int k, node *head) {
+
+    //number of element on which starts 'k-th to end' fragment
+    int n = length(head) - k;
+    //not sure what to do when n <= 0
+    //return what function found or do sth that shows there was an 'error'?
```

Comment:
You can choose how to handle bad input in whichever way you see fit.
For example, returning NULL is ok here, or throwing an exception.